src/Database.m

```diff
@@ -134,7 +134,7 @@ - (BOOL)initialiseDatabase {
     } else if (databaseVersion >= MA_Min_Supported_DB_Version) {
         NSAlert * alert = [[NSAlert alloc] init];
         [alert setMessageText:NSLocalizedString(@"Database Upgrade", nil)];
-        [alert setInformativeText:NSLocalizedString(@"Vienna must upgrade its database to the latest version. This may take a minute or so. We apologize for the inconveninece.", nil)];
+        [alert setInformativeText:NSLocalizedString(@"Database upgrade explanation text", @"Vienna must upgrade its database to the latest version. This may take a minute or so. We apologize for the inconvenience.")];
```

Comment:
Another benefit of this: given that this is just a typo, Crowdin will likely suggest the original string when translators view the corrected translation key. This can make it unnecessary to update the *.strings files manually, as it would be quicker to just go through the languages one by one.